index.bs

```diff
@@ -1824,11 +1827,37 @@ a numbered step. If outdented, it (today) is rendered as a bullet in the midst o
 
                 :   is present
                 ::  If <code>|pkOptions|.{{PublicKeyCredentialCreationOptions/rp}}.{{PublicKeyCredentialRpEntity/id}}</code> [=is not a
-                    registrable domain suffix of and is not equal to=] |effectiveDomain|, throw a "{{SecurityError}}" {{DOMException}}.
+                    registrable domain suffix of and is not equal to=] |effectiveDomain|, and if the client
+                
+                    <dl class="switch">
+                        :   supports [[#sctn-related-origins|related origin requests]]
+                        ::  1. Let |rpIdRequested| be the value of <code>|pkOptions|.{{PublicKeyCredentialCreationOptions/rp}}.{{PublicKeyCredentialRpEntity/id}}</code>.
+
+                            1. Let |maxLabels| be the number of maximum [=origin labels=] allowed by client policy.
+                        
+                            1. Fetch the well-known URL for the RP ID (<code>https://|rpIdRequested|/.well-known/webauthn-origins</code>).
```

Comment:
If the JSON served here already needs an `"origins"` key with the array of origins, what if this became the basis for a general WebAuthn configuration by changing the URL to `/.well-known/webauthn`?

Reply:
During WG @ 3/13: Could make collaboration with e.g. passkey management URLs trickier if WebAppSec WG wants to work on a similar well-known document.